overrides/partials/integrations/analytics/amplitude.html

```diff
@@ -35,6 +35,52 @@
         includeGclid: true
     });
 
+    function checkAuthenticatedUser() {
+        try {
+            const AMP_PREFIX = 'amp_';   
+            const currUserId = amplitude.getInstance().getUserId();
+            if (!currUserId) {
+                // Iterate through all amplitude cookies within amplitude domain
+                const amplitudeCookies = getCookies(AMP_PREFIX);
+                
+                for (let [cookieName, cookieValue] of amplitudeCookies.map((cookies) => cookies.split('='))) {
+                    const [deviceId, userId] = cookieValue.split('.');
+                    if (userId) {
+                        amplitude.getInstance().setUserId(decodeURIComponent(escape(window.atob(userId))));
+
+                        // If device id is found from other projects, use this device id
+                        if (deviceId) {
+                            amplitude.getInstance().setDeviceId(deviceId);
```

Comment:
Do we generate a device id when init? I guess we don't have to use device id from another project?